common/models/user.js

```diff
@@ -50,6 +54,23 @@ module.exports = function(User) {
   // see:
   // https://github.com/strongloop/loopback/issues/1137#issuecomment-109200135
   delete User.validations.email;
+  // Updated workaround for email validation
+  // see:
+  // https://github.com/strongloop/loopback/issues/1137#issuecomment-271437926
+  User.validate('email', function(err) {
+    if (this.email && !isEmail(this.email)) {
+      // Here err is a callback
+      // See loopback api docs loopback-datasource-juggler/#validatable-validate
+      debug(err());
```

Comment:
Okay, so I can surely see that this block as I mentioned in the comment above is doing nothing. Can we try testing this against some live data to be sure? We should just get rid of this altogether. Your change is robust as it is.